DxDispatch/CMakeLists.txt

```diff
@@ -62,8 +62,13 @@ add_onnxruntime_extensions_target(onnxruntime_extensions CACHE_PREFIX DXD)
 # Global Defines for CMAKE
 # ==============================================================================
 if(MSVC)
-    add_compile_options(/guard:cf /Qspectre /GS /sdl )
-    add_link_options(/GUARD:CF /DYNAMICBASE )
+    add_compile_options(/guard:cf /Qspectre /GS /sdl  /W3 )    
+    if( TARGET_ARCH STREQUAL ARM64 OR
+        TARGET_ARCH STREQUAL ARM)
+        add_link_options(/GUARD:CF /DYNAMICBASE)
+    else()
+        add_link_options(/GUARD:CF /DYNAMICBASE /CETCOMPAT)
```

Comment:
Is `/CETCOMPAT` a nop or error on ARM? If error, then I get it, but if it's NA/ignored on ARM, the fewer arch-specific branches we have, the easier it is to manage.

Reply:
unfortunately it is an error 

LINK : fatal error LNK1246: '/CETCOMPAT' not compatible with 'ARM64' target machine; link without '/CETCOMPAT'

Reply:
Gotcha. Then I'm changing your "Won't fix" to "Resolved", because there's nothing wrong to fix. :b